code.js

```diff
@@ -0,0 +1,368 @@
+//Class toDoList
+
+import { unCheckComplete } from './checkstatus';
+import {
+  addItem, removeItem, updateItem, clearComplted,
+} from './addremove';
+
+export default class ToDoList {
+  constructor() {
+    this.listItems = [];
+  }
+
+  setListItems(obj = null) {
+    localStorage.setItem('todolist', JSON.stringify(obj));
+  }
+
+  getListItmes=() => {
+    const obj = JSON.parse(localStorage.getItem('todolist'));
+    if (obj === null) {
+      this.listItems = [
+        { decription: 'Double-tap to edit', completed: false, index: 1 },
+        { decription: "Drag 'n drop to reorder your list", completed: false, index: 2 },
+        { decription: 'Manage all your lists in one place', completed: false, index: 3 },
+        { decription: 'Resync to clear out the old', completed: false, index: 4 },
+      ];
+      this.setListItems(this.listItems);
+    } else {
+      const mappedDataArray = [];
+      Object.keys(obj).map((key) => (mappedDataArray.push(obj[key])));
+
+      this.listItems = mappedDataArray;
+    }
+  }
+
+ displayItems=() => {
+   const listMain = document.createElement('ul');
+   const listItemTitle = document.createElement('li');
+   const iconRefresh = document.createElement('i');
+   const iconEnter = document.createElement('i');
+   iconRefresh.className = 'fas fa-sync iconRefresh';
+   iconEnter.className = 'fas fa-pen iconEnter';
+   listItemTitle.appendChild(document.createTextNode('To Do List'));
+   listItemTitle.appendChild(iconRefresh);
+   const listItemInput = document.createElement('li');
+   listItemInput.className = 'listItemInput';
+   const listInput = document.createElement('input');
+   listInput.type = 'text';
+   listInput.className = 'listInput';
+   listInput.id = 'listInput';
+   listInput.addEventListener('keyup', addItem, false);
```

Comment:
I think the code might look cleaner if you don't rewrite the default property. In this snippet, addEventListener's useCapture parameter is false by default. But if there is a point I missed, your return would make me very happy.

[addEventListener's useCapture parameter](https://www.w3schools.com/jsref/met_element_addeventlistener.asp)